TelemetryManager/Inc/TMCircularBuffer.hpp

```diff
@@ -76,6 +76,9 @@ class TMCircularBuffer : public CircularBuffer {
      * @return int The index of the current byte in the queue.
      */
     int currentIndex();
+
+    private: 
+    int index = 0;
```

Comment:
name it to be `index_`? If the index could never to be less than 0, then probably consider change the type to some unsigned data type.